install/0000_80_machine-config-operator_02_images.configmap.yaml

```diff
@@ -11,6 +11,7 @@ data:
       "etcd": "registry.svc.ci.openshift.org/openshift:etcd",
       "infraImage": "registry.svc.ci.openshift.org/openshift:pod",
       "kubeClientAgentImage": "registry.svc.ci.openshift.org/openshift:kube-client-agent",
+      "clusterEtcdOperatorImage": "",
```

Comment:
to be clear this is intentional and will be added in a separate PR when we enable the operator.